static/app/components/dynamicSampling/investigationRule.tsx

```diff
@@ -176,15 +176,14 @@ function InvestigationRuleCreationInternal(props: PropsInternal) {
   if (request.isLoading) {
     return null;
   }
-
-  if (request.error !== null) {
+  if (request.isError) {
     const errorResponse = t('Unable to fetch investigation rule');
     addErrorMessage(errorResponse);
     return null;
   }
 
   const rule = request.data;
-  const haveInvestigationRuleInProgress = rule !== null;
+  const haveInvestigationRuleInProgress = !!rule;
```

Comment:
I was a bit confused by this, why did you put `!!` isn't this equivalent to not putting it, or it's some JS magic?

Reply:
`!!x` just converts it to its boolean value, specifically here (`rule !==null`) I was explicitly testing against null which would not have worked if the rule was undefined or an empty string. 

I changed it because the mock version was not returning a null ( I think) and anyway it is safer, since my intention was not to check if the rule is precisely `null` but to check if I have a rule... `undefined` or `""` would also mean I don't have a rule.